Sources/SwiftKuerySQLite/SQLiteConnection.swift

```diff
@@ -459,3 +459,77 @@ public class SQLiteConnection: Connection {
         }
     }
 }
+
+class SQLiteColumnBuilder: ColumnCreator {
+    func buildColumn(for column: Column, using queryBuilder: QueryBuilder) -> String? {
+        guard let type = column.type else {
+            return nil
+        }
+
+        var result = column.name
+        let identifierQuoteCharacter = queryBuilder.substitutions[QueryBuilder.QuerySubstitutionNames.identifierQuoteCharacter.rawValue]
+        if !result.hasPrefix(identifierQuoteCharacter) {
+            result = identifierQuoteCharacter + result + identifierQuoteCharacter + " "
+        }
+
+        var typeString = type.create(queryBuilder: queryBuilder)
+        if let length = column.length {
+            typeString += "(\(length))"
+        }
+        if column.autoIncrement {
+            if column.isPrimaryKey && typeString == "integer" {
+                result += typeString + " PRIMARY KEY" + " AUTOINCREMENT"
+            } else {
+                //SQLite only allows autoincrement on integer PRIMARY KEY columns so return nil
+                return nil
```

Comment:
Same as above.

Reply:
The nil is handled in the calling function and an error thrown that identifies the column as having invalid attributes.